features/conversation/conversation-chat/conversation.nav.tsx

```diff
@@ -11,13 +12,27 @@ export type ConversationNavParams = {
 }
 
 export const ConversationScreenConfig = {
-  path: "/conversation",
+  path: "/conversation/:inboxId?",
   parse: {
-    topic: decodeURIComponent,
+    inboxId: (value: string | undefined) => {
+      if (!value) return undefined;
+      
+      const inboxId = decodeURIComponent(value) as IXmtpInboxId;
+      logger.info(`Parsing inboxId from URL: ${inboxId}`);
+      return inboxId;
+    },
+    composerTextPrefill: (value: string | undefined) => {
+      if (!value) return undefined;
+      
+      const text = decodeURIComponent(value);
+      logger.info(`Parsing composerTextPrefill from URL: ${text}`);
+      return text;
+    }
   },
   stringify: {
-    topic: encodeURIComponent,
-  },
+    inboxId: (value: IXmtpInboxId | undefined) => value ? encodeURIComponent(String(value)) : "",
+    composerTextPrefill: (value: string | undefined) => value ? encodeURIComponent(value) : "",
+  }
```

Comment:
Not sure, but I think this might be overkill since we're not really formating the value received no?